SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -291,39 +271,30 @@ bool SpatialEventTracer::IsStackEmpty() const
 	return SpanIdStack.Num() == 0;
 }
 
-void SpatialEventTracer::AddLatentPropertyUpdateSpanId(const TWeakObjectPtr<UObject>& Object, const Trace_SpanId& SpanId)
+void SpatialEventTracer::AddLatentPropertyUpdateSpanId(const TWeakObjectPtr<UObject>& Object, const FSpatialGDKSpanId& SpanId)
 {
-	if (!IsEnabled())
-	{
-		return;
-	}
-
-	Trace_SpanId* ExistingSpanId = ObjectSpanIdStacks.Find(Object);
+	FSpatialGDKSpanId* ExistingSpanId = ObjectSpanIdStacks.Find(Object);
 	if (ExistingSpanId == nullptr)
 	{
 		ObjectSpanIdStacks.Add(Object, SpanId);
 	}
 	else
 	{
-		Trace_SpanId MergeCauses[2] = { SpanId, *ExistingSpanId };
-		*ExistingSpanId = CreateSpan(MergeCauses, 2).GetValue();
+		FSpatialGDKSpanId CauseSpanIds[2] = { SpanId, *ExistingSpanId };
+		*ExistingSpanId =
+			TraceEvent(FSpatialTraceEventBuilder::CreateObjectPropertyComponentUpdate(Object.Get()), CauseSpanIds->GetConstId(), 2);
```

Comment:
Seems weird, maybe just 
```suggestion
			TraceEvent(FSpatialTraceEventBuilder::CreateObjectPropertyComponentUpdate(Object.Get()), CauseSpanIds, 2);
```

Reply:
Computer say no. So the TraceEvent takes a Trace_SpanIdType* which is uint8*. Can't pass in FSpatialGDKSpanId*

Reply:
Ah right, would need reinterpret_cast - really I don't mind but it seems weird to use a function to indicate we are accessing offset 0 of this memory